src/components/Planning/PlanningGroupsTable/PlanningGroupsTableRow.tsx

```diff
@@ -29,13 +54,15 @@ interface IPlanningProjectsTableProps {
 /**
  * We're only mapping the project name here for now since the values aren't yet implemented
  */
-const PlanningGroupsTableRow: FC<IPlanningProjectsTableProps> = ({
+const PlanningGroupsTableRow: FC<IPlanningGroupsTableRowProps> = ({
```

Comment:
onClick={(e) => onProjectMenuClick(project.id, e as unknown as MouseEvent)} move this anonymous function into const onClick = useCallback((e) => ...., [project])